### PR TITLE
custom-tabular-bq-managed-dataset.ipynb: Remove preprocessing

### PR DESCRIPTION
**Pull request by ivanmkc**
- [x] Removed pre-processing (normalization)

This simplifies the code but reduces accuracy.

### CODE REVIEW

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 1]
```diff
@@ -346,26 +346,6 @@
         "! gsutil mb -l $REGION -p $PROJECT_ID $BUCKET_URI"
```
Comment (ivanmkc) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <pre class="ql-syntax" spellcheck="false">Line #49.        return bq_client.list_rows(table).to_dataframe()
</pre><p>I tried using a <code>SELECT *</code> query instead but that requires elevated privileges (i.e. create) when all we should need are read privileges so I kept the current code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

**Review by gericdong — APPROVED** [review]
LGTM

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb (hunk 1 above)
Comment (kramachandran) [reply notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <p>Elevated compared to what? This looks to be a standard permission of GCE SAs</p>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <pre class="ql-syntax" spellcheck="false">Line #20.    df_numeric = (df_numeric - df_numeric.mean()) / df_numeric.std()
</pre><p>Shouldn't this normalization happen after the test is set is split from the training set</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <pre class="ql-syntax" spellcheck="false">Line #23.    # Convert categorical columns to numeric
</pre><p>I would have probably used  get_dummies() instead of factorize here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <p>I think a question someone could reasonably have is why am I writing this back to BQ? Maybe add some text justifying? Or should we write it back to BQ? Why not write it out GCS and then create a tabluar datasets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <p>I wonder if this batch of code should be closer to the actual invocation of the job.  It's a little hard to remember what you set all the parameters to up here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <p>Regarding the first warning.  Aren't we still normalizing the inputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <pre class="ql-syntax" spellcheck="false">Line #56.    def convert_dataframe_to_dataset(
</pre><p>I'd remove this function and just inline this code. Looks like you only call the function once so n need to have it as a function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <pre class="ql-syntax" spellcheck="false">Line #72.        y_train = tf.keras.utils.to_categorical(y_train, num_classes=num_species)
</pre><p>We are re-doing the categorization from above again. But using Keras. Any particular reason? Why not at least do it the same way?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (kramachandran) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:346]: <pre class="ql-syntax" spellcheck="false">Line #16.        args=CMDARGS,
</pre><p>Should bring the definition of CMDArgs down here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>
